src/explore-education-statistics-admin/src/components/Page.tsx

```diff
@@ -40,6 +47,47 @@ const Page = ({
 
       <PageHeader wide={wide} />
 
+      {showBanner && (
```

Comment:
Should this always be visible? This will also be shown to non-DfE pre-release users, which I suspect isn't intended behaviour.

Reply:
Good point, didn't see that!